tests/test_images/stride_align1/LZ4/test_A1_LZ4_align1.c

```diff
@@ -45,14 +45,14 @@ uint8_t test_A1_LZ4_align1_map[] = {
 
 };
 
-const lv_img_dsc_t test_A1_LZ4_align1 = {
+const lv_image_dsc_t test_A1_LZ4_align1 = {
   .header.magic = LV_IMAGE_HEADER_MAGIC,
   .header.cf = LV_COLOR_FORMAT_A1,
   .header.flags = 0 | LV_IMAGE_FLAGS_COMPRESSED,
   .header.w = 71,
   .header.h = 60,
   .header.stride = 16,
-  .data_size = 347,
+  .data_size = sizeof(test_A1_LZ4_align1_map),
```

Comment:
These should be updated in a separate PR.  I can help with reverting if needed.

Reply:
I'm not sure what to do with this. I think these were generated by the test script.